config/initializers/filter_parameter_logging.rb

```diff
@@ -4,5 +4,5 @@
 # Use this to limit dissemination of sensitive information.
 # See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += [
-  :passw, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
+  :passw, :email, :secret, :token, :_key, :crypt, :salt, :certificate, :otp, :ssn
```

Comment:
This will filter fields containing "email" in the inspect output which is relevant for the admin log which calls inspect on comments. Maybe we want to revert this?

Reply:
If we do need the info and access the field directly, e.g. `comment.email`, does it get filtered? If not, then filtering it out from `.inspect` output should be fine.

Reply:
With https://github.com/otwcode/otwarchive/pull/5303#issuecomment-3172475309, let's leave a TODO here to remove `email` once we've cleaned up the use of `inspect` in `AdminActivity.log_action`.

Reply:
Sorry, I misspoke... I meant leave `email` out for now and re-add it later.

But since the logs are only for user comments, you're right that we don't care. This is fine as is.